### ISSUE TITLE
StopBannerAutoRefresh does not work as expected

### ISSUE THREAD

**Issue by abogarsukov-braingames** [AppLovin/AppLovin-MAX-Unity-Plugin#347]
### MAX Plugin Version

6.2.1

### Unity Version

2021.3.28

### Device/Platform Info

Both iOS and Android, any device

### Current Behavior

After the `StopBannerAutoRefresh` call, MAX eventually starts auto-loading banners again. This results in OnBannerLoaded/OnBannerLoadFailed callbacks triggered without calling `LoadBanner`.

### Expected Behavior

StopBannerAutoRefresh is supposed to stop MAX from auto-loading banners. 

### How to Reproduce

There are no exact steps, it just happens eventually. Usually it starts after failing loading a banner. When it started, it continues till the end of the session.

### Additional Info

We disable MAX auto-refresh to manage banner loading manually. We expect to receive exactly one OnBannerLoaded/OnBannerLoadFailed callback for each `LoadBanner` call. But eventually we start getting unmatched callbacks. We experience the same problem for MRECs. Here's logcat trace of a session with this issue:
[applovin-banners-refresh.log](https://github.com/AppLovin/AppLovin-MAX-Unity-Plugin/files/14693293/applovin-banners-refresh.log)

**Comment by santoshbagadi** [comment]
From the logs, we've not been able to tell what is really causing the refresh to restart. 

Are you noticing the issue consistently on all sessions?

Is the issue happening on the current live version of the app?

**Comment by abogarsukov-braingames** [comment]
Yes we observe the issue in production. It's not reproducible on every session, but it happens often. I am able to reproduce it (sometimes) on a test application containing only AppLovin SDK and our wrapper code.

**Comment by santoshbagadi** [comment]
Would you mind sharing the test application for us to be able to investigate further?

**Comment by abogarsukov-braingames** [comment]
I will as soon as I have the stable repro.
